### PR TITLE
Fixes #13477 - Correct filter name in redirect after bulk edit

### LINKED ISSUE #13477: Bulk import with updates returns empty page - Wrong output filter
### NetBox version

v3.5.6

### Python version

3.8

### Steps to Reproduce

1 - Prepare a bulk import CSV, with ip_address, vrf, status, dns_name and description along with id, so already existing object would be updated.
2- Using WebUI, proceed with import at /ipam/ip-addresses/import/.
3- Notice that resulting URL is using "/?created_by_request=UID_HERE" 

### Expected Behavior

A green toast notification saying all good, and the results being displayed on the page, like this: 
image

### Observed Behavior

A green toast notification saying all good, and an empty page saying "No IP addresses found":
image

### PR DESCRIPTION
**Pull request by kkthxbye-code**
### Fixes: #13477

Makes create_and_update_objects return an extra bool indicating if the operation was creation or updating. Uses this bool to choose the correct filter.

### CODE REVIEW

**Review by jeremystretch — CHANGES_REQUESTED** [review]

File: netbox/netbox/views/generic/bulk_views.py [hunk 1]
```diff
@@ -418,7 +419,7 @@ def create_and_update_objects(self, form, request):
 
                 raise ValidationError("")
 
-        return saved_objects
+        return saved_objects, is_update
```
[the remark this answers is not in the document]
Comment (kkthxbye-code) [reply netbox/netbox/views/generic/bulk_views.py:422]: I wasn't even aware that both were supported in the same inport, I'll take another look at it.

[the remark this answers is not in the document]
Comment (jeremystretch) [reply netbox/netbox/views/generic/bulk_views.py:422]: Well, I only know it's _possible_. I don't know if it's _supported_. :slightly_smiling_face:

[the remark this answers is not in the document]
Comment (kkthxbye-code) [reply netbox/netbox/views/generic/bulk_views.py]: I added `modified_by_request` and changed the redirect to use this filter. It finds both update and create changes. I don't think  deletions make any sense as filtering an query looking for deleted objects is bound to always return nothing 😄

**Comment by jeremystretch** [comment]
Thanks @kkthxbye-code!

**Pull request merged**